src/components/DefineCompany/FolioNumber.vue

```diff
@@ -1,28 +1,28 @@
 <template>
   <div id="folio-number">
     <v-layout row v-if="!isEditing" id='folio-number-read-only'>
-        <v-flex md4>
-          <label><strong>Folio Information</strong></label>
-        </v-flex>
-        <v-flex md4>
-          <div><label><strong>Folio Number</strong></label></div>
-          <div id="lbl-folio-number">{{ !!folioNumber ? folioNumber : 'Not entered' }}</div>
-        </v-flex>
+      <v-flex md4>
+        <label><strong>Folio Information</strong></label>
+      </v-flex>
+      <v-flex md4>
+        <div><label><strong>Folio Number</strong></label></div>
```

Comment:
Just curious, does it need to be a <div> if it is a <label>?

Reply:
Nope! Existing code that I simply de-indented. Will fix in next commit.